ext/imap/php_imap.c

```diff
@@ -4242,6 +4236,40 @@ static zend_string* _php_imap_parse_address (ADDRESS *address_list, zval *paddre
 }
 /* }}} */
 
+static void php_imap_update_property_with_parsed_full_address(
+	zval *z_object, ADDRESS *address,
+	const char *str_address_property, size_t str_address_property_len,
+	const char *parts_address_property, size_t parts_address_property_len
+)
+{
+	zval address_parts;
+	zend_string *full_address = NULL;
+
+	array_init(&address_parts);
+	full_address = _php_imap_parse_address(address, &address_parts);
+	if (full_address) {
+		zend_update_property_str(
+			Z_OBJCE_P(z_object), Z_OBJ_P(z_object),
+			str_address_property, str_address_property_len,
+			full_address
+		);
+		zend_string_release(full_address);
```

Comment:
I wonder if this can be `zend_string_release_ex(full_address, false);` (and similar in other places)...

Reply:
Not actually important though.